app/models/group_resource.rb

```diff
@@ -0,0 +1,7 @@
+class GroupResource < ActiveRecord::Base
```

Comment:
GroupResource is the join table that joins groups with resources (cookbooks or tools).